inspire/dojson/institutions/schemas/insitutions-0.0.1.json

```diff
@@ -8,13 +8,53 @@
             "type": "string"
         },
         "address": {
-            "description": "Street etc, city with postal code + additions (native language) FIXME: isn't postal code redundant?",
-            "title": "Address",
-            "type": "string"
-        },
-        "city": {
-            "title": "City",
-            "type": "string"
+            "items": {
+                "properties": {
+                    "address": {
+                        "description": "Street etc, city with postal code + additions (native language) FIXME: isn't postal code redundant?",
+                        "title": "Address",
+                        "type": "string"
+                    },
+                    "city": {
+                        "title": "City",
+                        "type": "string"
+                    },
+                    "state_province": {
+                        "title": "State or Province",
+                        "type": "string"
+                    },
+                    "country": {
+                        "description": "FIXME can't we derive this from country_code?",
+                        "enum": [
+                            "does",
+                            "it",
+                            "exist?"
+                        ],
+                        "string": "Country name",
+                        "type": "string"
+                    },
+                    "postal_code": {
+                        "title": "Postal code",
+                        "type": "string"
+                    },
+                    "country_code": {
+                        "enum": [
+                            "let's",
+                            "take",
+                            "the",
+                            "enum",
+                            "from",
+                            "iso"
+                        ],
+                        "title": "Country code",
+                        "type": "string"
+                    },
+                },
+                "title": "Address",
+                "type": "object"
+            },
+            "title": "Addresses",
+            "type": "array",
```

Comment:
Hi @jalavik, when I created the model, I though of keeping it as flat as possible in order to simplify coding and configuration. So e.g. if the city is only one I had decided to put it on the root level directly. Why have you decided to structure them in a deeper way?